app/definitions/IMessage.ts

```diff
@@ -147,7 +147,10 @@ export interface IMessage extends IMessageFromServer {
 	editedAt?: string | Date;
 }
 
-export type TMessageModel = IMessage & Model;
+export type TMessageModel = IMessage &
+	Model & {
+		asPlain?: () => IMessage;
```

Comment:
I need to type `asPlain` as optional because if I type it as obligatory, other places were complain about it